lib/xero_gateway/oauth.rb

```diff
@@ -38,12 +40,12 @@ def consumer
     end
 
     def request_token(params = {})
-      @request_token ||= consumer.get_request_token(params)
+      @request_token ||= consumer.get_request_token(params, nil, @base_headers)
```

Comment:
Took me a while to track down the reason for the `nil` parameter here.  https://github.com/oauth-xx/oauth-ruby/blob/master/lib/oauth/consumer.rb#L353-L363

I think it's worth a comment at least to explain the argument list.